### PR TITLE
DOC: Add links and tighten Contribute page

### PR DESCRIPTION
**Pull request by bjnath**
Adds links to the bulleted list of projects to make it clear they're
explained further below.

Adds section on reviewing PRs based on @anirudh2290's recent email.

Tighter wording throughout.

Old and new versions:

old:
image

new:
image

### CODE REVIEW

**Comment by bjnath** [comment]
Current version (after minor changes in b17ba61):

image

File: content/en/contribute.md [hunk 1]
```diff
@@ -3,74 +3,121 @@ title: Contribute to NumPy
...
+and welcoming environment.
+
+### Writing code
+
+Programmers, this
+[guide](https://numpy.org/devdocs/dev/index.html#development-process-summary)
+explains how to contribute to the codebase.
+
+### Reviewing pull requests
+Even inexperienced contributors can help us deal with our surging PR count
```
Comment (rgommers) [inline content/en/contribute.md:53]: This message I'd rather tweak a bit. Inexperienced developers trying to move PRs forward often add more noise and overhead than being helpful.

File: content/en/contribute.md [hunk 2]
```diff
@@ -3,74 +3,121 @@ title: Contribute to NumPy
...
-Between community calls, our Twitter account, improving our documentation on processes and community topics, organizing sprints, a newsletter, and perhaps in the future a blog, there is a lot of organizing and outreach to do. We'd love to get more people involved in this important aspect of the project!
+Through community contact we share our work more widely and learn where we're
+falling short. We'd love to get more people involved in efforts like our
+[Twitter](https://twitter.com/numpy_team) account, organizing NumPy [code
+sprints](https://scisprints.github.io/), a newsletter, and perhaps a blog.
+
+### Fundraising
 
+For most of its history NumPy was all-volunteer, but as the user base grew,
+paid staffers were added to ensure the project stayed healthy. [This SciPy'19
```
Comment (rgommers) [inline content/en/contribute.md:119]: I'm not sure about the "paid staffers" and past tense here. It makes it sound like a solved problem, which it very much is not. The main source of funding was the BIDS grant, and that's running out in October. And even with that, we were still short on maintenance power to actually reduce the backlog of PRs for example. Could you try to rephrase?

File: content/en/contribute.md (hunk 1 above)
Comment (bjnath) [reply content/en/contribute.md]: Indeed! Revised version clarifies the baseline skills.

File: content/en/contribute.md (hunk 2 above)
Comment (bjnath) [reply content/en/contribute.md]: Thanks for clarifying this. Revised version makes clear that funding needs are ongoing.

**Comment by bjnath** [comment]
b31325b addresses @rgommers comments, also:
    * Removes bullets that have no links below
    * Uses mailto: markup on links with a mailing address
    * Small wording changes in graphic design and translator sections

New page:
  
image

**Comment by bjnath** [comment]
Page now:

image

**Comment by bjnath** [comment]
> [Thanks @bjnath, I like this rewrite...I'll come back to this soon](https://github.com/numpy/numpy.org/pull/314#pullrequestreview-423468417)

@rgommers, is there an opportunity to revisit this PR? I've rewritten sections you had comments about.

**Review by rgommers — APPROVED** [review]
Resolved one merge conflict and added a minor tweak. In it goes. Apologies for the long delay @bjnath.

**Pull request merged**